RESULTS.md

```diff
@@ -0,0 +1,61 @@
+Product Test result Summary
+---
+
+Product test results from be2fc0baeda69cad7480e29aa6714b2e5d916c8e tested on 2021-09-30
+
+| Product                           | Passing Jobs  | Failing Jobs  | Skipped Jobs  |
+|-----------------------------------|---------------|---------------|---------------|
+| rocky-boot-iso-x86_64-*           | 0             | 2             | 0             |
+| rocky-minimal-iso-x86_64-*        | 2             | 0             | 0             |
+| rocky-dvd-iso-x86_64-*            | 1             | 6             | 16            |
+| rocky-universal-x86_64-*          | 8             | 24            | 0             |
+
+
+Product test commands

```

Comment:
Are there any other configurations that should be here, such as `PACKAGE_SET=server` as mentioned in 
[OpenQA-Fedora-Installation notes](https://github.com/rocky-linux/OpenQA-Fedora-Installation/blob/develop/NOTES.md)?

Reply:
To make dvd-iso fully working you actually need a `PACKAGE_SET`, either `minimal`, `server` or `virtualization-host` is available right now 👍🏻